django_rich_logging/logging.py

```diff
@@ -47,6 +48,7 @@ def emit(self, record: logging.LogRecord) -> None:
             request = record.args[0]
             status = record.args[1]
             size = ""
+            time = self.formatter.formatTime(record, datefmt=self.formatter.datefmt)
```

Comment:
I think I need to handle when `self.formatter` is None 🤔

I can probably set a default formatter in `DjangoRequestHandler.__init__` if it's None.

Reply:
Added in [39c5407](https://github.com/adamghill/django-rich-logging/pull/5/commits/39c5407a7d348e92bfdfbcc4c543d17a04cf0aa2)